src/Symfony/Bridge/Twig/Extension/IntlExtension.php

```diff
@@ -0,0 +1,317 @@
+            new \Twig_SimpleFilter('localized_language', array($this, 'getLocalizedLanguage')),
+            new \Twig_SimpleFilter('localized_locale', array($this, 'getLocalizedLocale')),
+            new \Twig_SimpleFilter('localized_number', array($this, 'getLocalizedNumber')),
+        );
+    }
+
+    /**
+     * Returns a localized country name for a given code.
+     *
+     * @param string      $country
+     * @param string|null $displayLocale
+     *
+     * @return null|string
+     */
+    public function getLocalizedCountry($country, $displayLocale = null)
+    {
+        return Intl::getRegionBundle()->getCountryName((string) $country, $displayLocale ?: $this->defaultLocale);
+    }
+
+    /**
+     * Returns a localized currency amount for given number and currency code.
+     *
+     * @param mixed       $amount
+     * @param string      $currency
+     * @param string|null $displayLocale
+     *
+     * @return null|string
+     */
+    public function getLocalizedCurrency($amount, $currency, $displayLocale = null)
+    {
+        $formatter = $this->createNumberFormatter(
+            $this->numberFormatterClass,
+            $displayLocale ?: $this->defaultLocale,
+            'currency'
+        );
+
+        return $formatter->formatCurrency((float) $amount, $currency);    }
+
+    /**
+     * Returns a localized currency amount for a given number and currency code.
+     *
+     * @param string      $currency
+     * @param string|null $displayLocale
+     *
+     * @return null|string
+     */
+    public function getLocalizedCurrencyName($currency, $displayLocale = null)
+    {
+        return Intl::getCurrencyBundle()->getCurrencyName((string) $currency, $displayLocale ?: $this->defaultLocale);
+    }
+
+    /**
+     * Returns a currency symbol for a given code.
+     *
+     * @param string      $currency
+     * @param string|null $displayLocale
+     *
+     * @return null|string
+     */
+    public function getCurrencySymbol($currency, $displayLocale = null)
+    {
+        return Intl::getCurrencyBundle()->getCurrencySymbol((string) $currency, $displayLocale ?: $this->defaultLocale);
+    }
+
+    /**
+     * Returns a date formatted for a given locale, and options.
+     *
+     * @param \DateTimeInterface|string $date
+     * @param string                    $dateFormat
+     * @param string                    $timeFormat
+     * @param string|null               $displayLocale
+     * @param \DateTimeZone|string|null $timezone
+     * @param string|null               $format
+     * @param string                    $calendar
+     *
+     * @return bool|string
+     *
+     * @throws \Exception|\InvalidArgumentException If the date or a format is invalid
+     */
+    public function getLocalizedDate($date, $dateFormat = 'medium', $timeFormat = 'medium', $displayLocale = null, $timezone = null, $format = null, $calendar = 'gregorian')
```

Comment:
-1 for this method. Twig_Extensions_Extension_Intl already provides the feature (and reuses the Twig logic to handle the display timezone instead of hardcoding the default to UTC, which is much better)